data/artifacts.js

```diff
@@ -1 +1,258 @@
\ No newline at end of file
+export default [
+  {
+    name: 'Divine Robe',
+    cost: 'mana-green-2',
+    color: 'white',
+    attack: '0',
+    defense: '0',
+    ability_1: 'monsoon',
```

Comment:
perhaps we should use an array of abilities?

Reply:
I agree - this was another hack to continue pulling card data from the google sheet. I'm a fan of the sheet because as we get closer to a playable game, it's a nice interface for non-technical folks to contribute to rules/mechanics. If there's a nice way to pull an array from the sheet, that would be preferable.

Reply:
can we use an [additional layer of indirection](https://stackoverflow.com/q/18003544) here?

e.g. add a data transformer to ingest a range of columns from the sheet (e.g. F-H) into an array in our data structure.